Lib/test/test_textwrap.py

```diff
@@ -475,6 +475,30 @@ def test_narrow_non_breaking_space(self):
                          'non-breaking\N{NARROW NO-BREAK SPACE}space.'],
                         break_on_hyphens=False)
 
+    def test_wrap_stability(self):
+        def wrap(s):
+            return fill(s, width=30)
+
+        # original is 31 characters long:
+        #           0        1         2         3
+        #           1234567890123456789012345678901
+        original = "xxxx xxxx xxxx xxxx xxxx.  xxxx"
+        wrapped = wrap(original)
+        wrapped2 = wrap(wrapped)
+        self.assertEqual(wrapped, wrapped2)
+
+    def test_wrap_stability_with_fix_sentence_endings(self):
```

Comment:
Seems this case is already covered by test_fix_sentence_endings. test_fix_sentence_endings fails if disable the correction for fix_sentence_endings.